server/cluster_worker_test.go

```diff
@@ -732,7 +732,14 @@ func (s *testClusterWorkerSuite) TestSplit(c *C) {
 		{"e", "d", "", "f"},
 	}
 
-	for _, t := range tbl {
+	var firstRegionStartKey, firstRegionEndKey string
+	for i, t := range tbl {
+		// Get first region start and end key.
+		if i == 0 {
+			firstRegionStartKey = t.startKey
+			firstRegionEndKey = t.splitKey
+		}
+
```

Comment:
I think `firstRegionStartKey, firstRegionEndKey := tbl[0].startKey, tbl[0].splitKey` would be more straight forward.